src/applications/_mock-form/helpers.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export const hasDirectDeposit = formData =>
+  formData.viewNoDirectDeposit !== true;
+
+export const directDepositWarning = (
+  <div className="pension-dd-warning">
+    The Department of Treasury requires all federal benefit payments be made by
+    electronic funds transfer (EFT), also called direct deposit. If you don’t
+    have a bank account, you must get your payment through Direct Express Debit
+    MasterCard. To request a Direct Express Debit MasterCard you must apply at{' '}
+    <a
+      href="http://www.usdirectexpress.com"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
```

Comment:
```suggestion
    aria-label="usdirectexpress.com, opens in new window"
    >
```

Reply:
I ended up removing the `helpers.js` file since it was for the custom direct deposit component. I switched to using the platform definition for this - we'll need to eventually update the complex form template